BDArmory/FX/FireFX.cs

```diff
@@ -187,7 +187,7 @@ void FixedUpdate()
                 return;
             }
             if (vacuum) transform.rotation = Quaternion.FromToRotation(Vector3.up, parentPart.vessel.obt_velocity.normalized);
-            else transform.rotation = Quaternion.FromToRotation(Vector3.up, -FlightGlobals.getGeeForceAtPosition(transform.position));
+            else transform.rotation = Quaternion.FromToRotation(Vector3.up, VectorUtils.GetUpDirection(transform.position));
```

Comment:
This could use `parentPart.vessel.up`. It's not going to be any different to the up direction at the part.